README.md

```diff
@@ -6,11 +6,35 @@
 For updates and history of development please see [this page](https://github.com/amiaopensource/audiorecorder/blob/master/history.md).
 
 ## Installation:
-Can be installed on macOS via Homebrew with the following command: 
+
+audiorecorder can be installed via [homebrew](https://brew.sh/) on macOS and [linuxbrew](http://linuxbrew.sh/) on Linux.
+
+__MacOS Installation__:
+Audiorecorder can be installed via Homebrew with the following command:
 
 `brew tap amiaopensource/amiaos && brew install audiorecorder`.
 
-Linuxbrew install testing is currently being conducted.
+__Linux Installation__ : (Tested in Ubuntu 16.04)
+
+The following division of commands seeks to install up-to-date versions of audiorecorder dependencies while avoiding conflicts.
+
+__Standard package manager commands:__
+
+`sudo apt-get install sox`
+
+`sudo apt-get install libsdl2-2.0`
+
+Install current [MPV](https://mpv.io/installation/) ppa, then `sudo apt-get install mpv`
+
+__linuxbrew commands:__
+
+`brew install dialog`
+
+`brew install ffmpeg --with-freetype --with-sdl2`
```

Comment:
I guess, you should say somewhere that this is necessary if not yet done with these parameters. Otherwise it gives an error message, saying that is already installed…